jobs/ci-kubernetes-bazel-test.sh

```diff
@@ -25,6 +25,6 @@ bazel clean --expunge
 make bazel-test && rc=$? || rc=$?
 
 # Coalesce test results into one file for upload.
-"$(dirname "${0}")/../images/pull_kubernetes_bazel/coalesce.py"
+"$(dirname "${0}")/../images/bazelbuild/coalesce.py"
```

Comment:
I'm going to undo my rename change, but I'm going to stick coalesce.py in the image, which will hopefully make this rename easier in a future change.